src/coreComponents/python/modules/geosx_mesh_tools_package/geosx_mesh_tools/abaqus_converter.py

```diff
@@ -8,13 +8,16 @@
 
 def convert_abaqus_to_gmsh(input_mesh, output_mesh, logger=None):
     """
-    @brief Convert an abaqus mesh to gmsh 2 format, preserving nodeset information.
-    @details If the code encounters any issues with region/element indices,
-             the conversion will attempt to continue, with errors
-             indicated by -1 values in the output file.
-    @param input_mesh path of the input abaqus file
-    @param output_mesh path of the output gmsh file
-    @param logger an instance of logging.Logger
+    Convert an abaqus mesh to gmsh 2 format, preserving nodeset information.
+
+    If the code encounters any issues with region/element indices,
+    the conversion will attempt to continue, with errors
+    indicated by -1 values in the output file.
+
+    Args:
+        input_mesh (str): path of the input abaqus file
+        output_mesh (str): path of the output gmsh file
+        logger (logging.Logger): an instance of logging.Logger
```

Comment:
So could se star using type hints?

Reply:
Sounds good to me.  With this change, do we want to use the backwards-compatible syntax, or the in-place syntax?
https://peps.python.org/pep-0484/#suggested-syntax-for-python-2-7-and-straddling-code

Reply:
I don't think that we do at this point

Reply:
Maybe it's the first move then 🤣 ?